support/awsutil/sg.go

```diff
@@ -227,3 +229,50 @@ func GetSecurityGroupById(ec2Client ec2iface.EC2API, id string) (*ec2.SecurityGr
 	}
 	return describeSGResult.SecurityGroups[0], nil
 }
+
+func UpdateResourceTags(ec2Client ec2iface.EC2API, resourceID string, create, remove map[string]string) error {
+	// If we have anything to create or update
+	if len(create) > 0 {
+		createTagsInput := MapToEC2Tags(create)
+		input := &ec2.CreateTagsInput{
+			Resources: []*string{aws.String(resourceID)},
+			Tags:      createTagsInput,
+		}
+
+		// Create/Update tags in AWS.
+		if _, err := ec2Client.CreateTags(input); err != nil {
+			return errors.Wrapf(err, "failed to create tags for resource %q: %+v", resourceID, create)
+		}
+	}
+
+	// If we have anything to remove
+	if len(remove) > 0 {
+		removeTagsInput := MapToEC2Tags(remove)
+		// Create the DeleteTags input
+		input := &ec2.DeleteTagsInput{
+			Resources: []*string{aws.String(resourceID)},
+			Tags:      removeTagsInput,
+		}
+
+		// Delete tags in AWS.
+		if _, err := ec2Client.DeleteTags(input); err != nil {
```

Comment:
Same as above. If the tag you're trying to delete is not there anymore and an error is returned, we should likely ignore that error.

Reply:
This also should not return an error on non-existent tags. according to AWS docs :)